.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+> Hi there! Thanks for submitting a pull request to Concourse! To help us review your PR,
+> please fill in the following information. Feel free to delete any sections not applicable to your pull request.
+
+# Existing Issue
+> Is there an existing issue related to this PR? Fill in the issue number as follows: Fixes #1234.
+
+Fixes # .
+
+# Issue Description
+> No issue number? Please give us a few lines of context describing the need for this PR:
+
+
+# Changes proposed in this pull request
+> What are the changes included in this PR? Feel free to add as many lines as needed below:
+
+* 
+* 
+*
+
+
+# Has the documentation been updated to reflect these changes?
+- [ ] Yes
+- [ ] No
+
+# Have the release notes been updated to reflect these changes?
+- [ ] Yes
+- [ ] No
+
```

Comment:
I remember talking about adding a checklist for reviewers as well, so if the reviewer changes the next person would know where they left off.

wdyt?

Reply:
What should we add in the checklist for reviewers? I remember we talked about performing acceptance and recording that somehow, but somehow one check mark for `performed acceptance` doesn't seem enough :/

Reply:
Gave it a try, see if the new checklist makes sense

Reply:
looks great!